### DISCUSSION TITLE
By default, rofi run can't deal with applications with an space in its name

### DISCUSSION THREAD

**Discussion by 2403772980ygy** [davatorium/rofi#1945] [Q&A]
I recently used Surge XT, whose name is 'Surge XT', and it seems that rofi tends to treat this as running Surge with XT as its argument. How do I correct this behaviour?

**Comment by DaveDavenport** [comment]
good find. Fixed defaults it in git. 
realize now my fix might break other things.. will look at it when I have more time.

**Reply by 2403772980ygy** [reply]
Actually I thought it might be my misuse, good to find out.

**Reply by DaveDavenport** [reply]
no its a valid issue.

**Reply by DaveDavenport** [reply]
better fix in git (I hope)
